### PR TITLE
Disable cat blocks scripts on april fools

### LINKED ISSUE #1896: disable cat blocks addon when it's april fools day
april fools day for 2021 currently seems to have catblocks planned. we should disable catblocks on our end to ensure ntohing wakcy happens

### PR DESCRIPTION
**Pull request by TheColaber**
**Resolves**

Resolves #1896

**Changes**

just `return`s

**Reason for changes**

So no conflicts with this year.

**Tests**

no but

### CODE REVIEW

**Comment by apple502j** [comment]
Can you make it so that it detects whether Scratch is running AF or not?

File: addons/cat-blocks/userscript.js [hunk 1]
```diff
@@ -1,4 +1,7 @@
 export default async function ({ addon, global, console }) {
+  const now = new Date().getTime() / 1000;
+  // April fools
+  if (now < 1617364800 && now > 1617192000) return;
```
Comment (WorldLanguages) [inline addons/cat-blocks/userscript.js:4]: We don't know when the ST will decide to enable/disable april fools, might not match our schedule

**Comment by WorldLanguages** [comment]
> Can you make it so that it detects whether Scratch is running AF or not?

@apple502j Can you suggest a way to do that?

**Comment by TheColaber** [comment]
I was thinking of checking if one of the blockly constants were changed but /shrug

**Comment by WorldLanguages** [comment]
After some testing, everything seems to work fine even if we run cat blocks above cat blocks.

**Pull request closed without merging**